src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -56,13 +59,21 @@ void draw_settings_dialog(HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     if (ImGui::BeginTabItem("Emulator")) {
         ImGui::PopStyleColor();
+        ImGui::Spacing();
+        ImGui::Combo("Log Level \nSelect your preferred log level.", &host.cfg.log_level, "Trace\0Debug\0Info\0Warning\0Error\0Critical\0Off\0");
         ImGui::Checkbox("Archive Log", &host.cfg.archive_log);
         ImGui::SameLine();
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Check the box to enable Archiving Log.");
         ImGui::Checkbox("Texture Cache", &host.cfg.texture_cache);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Uncheck the box to disable texture cache.");
+        ImGui::Spacing();
+        ImGui::PushItemWidth(400);
+        ImGui::InputTextWithHint("Set your Path Emulator \n(Reboot after change for apply)", "Write your path folder here", &host.cfg.pref_path);
+        ImGui::PopItemWidth();
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("For can use personal folder for Vita3K. \nCopy and past here your path Vita3K folder.");
```

Comment:
At least try to check your grammar, this is just unacceptable, we are writing software for general use.

```suggestion
            ImGui::SetTooltip("Store game data in your personal folder. \nAdd the path to the folder here.");
```